packages/styles/scss/patterns/blocks/content-group-pictograms/index.scss

```diff
@@ -6,5 +6,5 @@
 //
 
 @import '../../../globals/imports';
-@import 'pictogramarray';
-@import 'pictogramarray-item';
+@import '@carbon/grid/scss/grid';
```

Comment:
What we need from `@carbon/grid` is already included within `globals/imports`. Also, I don't think we want to include the whole grid at this time. Just the mixins.

Reply:
I've removed the grid import, thanks!